3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/IgluAdapter.scala

```diff
@@ -47,10 +48,50 @@ object IgluAdapter extends Adapter {
 
   // Tracker version for an Iglu-compatible webhook
   private val TrackerVersion = "com.snowplowanalytics.iglu-v1"
+  private val VendorName = "Iglu"
 
   // Create a simple formatter function
   private val IgluFormatter: FormatterFunc = buildFormatter() // For defaults
 
+    /**
+     *
+     * Converts a payload into a list of validated events
+     * Expects a valid json - returns a single failure if one is not present
+     *
+     * @param body json payload as POST'd by a webhook
+     * @param payload the rest of the payload details
+     * @return a list of validated events, successes will be the corresponding raw events
+     *         failures will contain a non empty list of the reason(s) for the particular event failing
+     */
+    private def payloadBodyToEvents(body: String, payload: CollectorPayload): List[Validated[RawEvent]] = {
```

Comment:
This function is returning a list of Validated RawEvents, however the actual body of the function is only ever going to return a single RawEvent in the List.  This should be changed to return just a `Validated[RawEvent]`.